### DISCUSSION TITLE
Is it a little cheeky to request support for the new neural2 voice?

### DISCUSSION THREAD

**Discussion by realIK17** [pgmichael/wavenet-for-chrome#48] [Ideas]
I have no programming skills so I have to use google's official node.js tutorial. But then I have to turn all my texts into one line and break them into chunks of 5000 characters by hand. So it would be great if this extension supports the new Neural 2 voices.

**Comment by 4357845798** [comment]
Definitely agree, this would be a lifesaver!

**Comment by pgmichael** [comment]
I don't actively maintain the extension anymore, but I'll take a look. If it's an easy addition I'll do it.

**Comment by IntrepidToad** [comment]
Yep, this extension is already awesome for productivity, and Neural2 voices would be even more amazing.

**Comment by pgmichael** [comment]
Just submitted the new version of the extension to chrome store! 🎈  Might take a day or two before the update goes live.

I must say however I was a bit disappointed with the quality of the `Neural2` voices. They seem almost the same as the `Wavenet` voices to me. Any use case in particular on why you'd want this voice over the previous `Wavenet` version?

Also added the `Studio` and `News` voices. The studio one is outstanding :)

**Reply by IntrepidToad** [reply]
Thank you so much for adding this!  I lean on screen readers a lot, so this is going to make my life tangibly better.

Just from testing the Neural2 demo, I found that Neural2-G does a lot to smooth over the computer monotone qualities of Wavenet-G.  It also is better at hiding those tonal "hitches" that happen whenever a new word is said.  I haven't come up with a better description word, but hitches are like that old Siri voice where every word feels like it is an excerpt from a speech (not far from the truth in that case), and nothing feels like it was actually said together in the same sentence.  So not only do individual words flow better in Neural2, but the overall sentence takes on a more cohesive rising or falling tone, making it more human.  The difference is not gigantic, but any steps towards making the voice more human make listening for long periods much easier.  Sometimes you can actually lose track of what's being said without proper use of tone, so it's an improvement for productivity.  

And those studio voices—I had no idea they existed until you pointed them out.  They're kinda awesome, I might actually make them my main voice if the demo is indicative of anything.  They seem better in most regards (tonality, clarity, "humanity") than neural2.  The breaths between words initially creeped me out, but I guess human is what I wanted, so this has to be even better by my criteria.

Edit: I tested the voices a little more and it seems like there's something wrong with Neural2, at least in the demo.  The first sentence of _A Tale of Two Cities_ consistently gets Neural2-G to go off the rails by the end, sounding like it's having a stroke.  I guess I'll have to test more once the update is live, but it's odd that Google's own demo would have such behavior.

**Comment by MEH-YAR** [comment]
Hello: The extension is great! Thank you! 
I have a question, is there a way to re-enable the keyboard shortcut to speak the text. Previously the keyboard shortcut to speak the text was (Command Shift and S) which was very helpful. Currently, there is only a keyboard shortcut to download mp3. 

Any solution? 
Thank you

**Reply by pgmichael** [reply]
Should still be available. Maybe you have another extension interfering. You can remove/change the shortcuts by going to this URL: chrome://extensions/shortcuts

**Reply by MEH-YAR** [reply]
Thank you, you are right! I added it using the URL you shared. Now it is working :)
